src/actions/LoginActions.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import request from 'superagent'
```

Comment:
I ran into some issues with request and webpack/babel that I couldn't manage to fix. That's why I switched to superagent